src/frontend/apps/standalone_site/package.json

```diff
@@ -38,6 +38,7 @@
     "build": "react-scripts build",
     "postbuild": "./postbuild.sh",
     "test": "cross-env CI=true TZ=UTC react-scripts test",
+    "test:CI": "cross-env CI=true TZ=UTC react-scripts test --watchAll=false --maxWorkers=2",
```

Comment:
```suggestion
```

Reply:
The rebase de "feature/anthony/create-classroom" will this.